src/components/Navigation/TopBar/Help/Feedback/Feedback.types.ts

```diff
@@ -0,0 +1,65 @@
+import { FileWithPath } from 'react-dropzone';
+
+import { ApiError, ServiceNowIncidentResponse } from '../../../../../api';
```

Comment:
Import can be shortened

Reply:
I thought there was some issue with using absolute path (src/api) in amplify-components, when getting other things from there.

Reply:
I think I fixed it, you can verify by trying to build and use😎

Reply:
It does work, nice! 👌